henson_amqp/__init__.py

```diff
@@ -131,6 +131,7 @@ def _begin_consuming(self):
                 durable=self.app.settings['AMQP_INBOUND_EXCHANGE_DURABLE'],
                 exchange_name=self.app.settings['AMQP_INBOUND_EXCHANGE'],
                 type_name=self.app.settings['AMQP_INBOUND_EXCHANGE_TYPE'],
+                arguments=self.app.settings['AMQP_INBOUND_EXCHANGE_KWARGS'],
```

Comment:
It looks like the other arguments are listed alphabetically. Should `arguments go first? (cc @jonafato)